### DISCUSSION TITLE
Powershell core in Linux Self-hosted agent

### DISCUSSION THREAD

**Discussion by MarcVanBelleghem** [whiteducksoftware/azure-devops-build-agents-community#7] [Q&A] [answered]
Hi, how can I install powershell core on Whiteduck Linux Scaleset agent to run powershell scripts from my pipeline?

**Comment by nmeisenzahl** [comment, marked as answer]
Hi @MarcVanBelleghem!

The linux-based Build Agent SCALABLE is intended to be generic and more flexible. It is designed to be used with containerized pipelines only. 

You can use the agent with the so-called [Container Jobs](https://learn.microsoft.com/azure/devops/pipelines/process/container-phases?view=azure-devops). Furthermore, you will find a generic pipeline example in our documentation [here](https://github.com/whiteducksoftware/azure-devops-build-agents-community/blob/main/docs/container-job.md).

We have also added a PowerShell example in our examples folder [here](https://github.com/whiteducksoftware/azure-devops-build-agents-community/blob/main/examples/container-job-powershell/container-job-ps.yaml).

Hope this helps you get started with our build agent!
